setup.py

```diff
@@ -16,6 +16,7 @@
 setup(
     name='spotify_dl',
     version=version,
+    python_requires='>=3',
```

Comment:
@iambibhas sorry for the late reply. I do mention Py3 is required, so feel better to enforce that